student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
```

Comment:
I would suggest you add default values for name and parent_permission
Example:
`name = "Unknown"`
`parent_permission = true`